types/three/three-core.d.ts

```diff
@@ -7116,6 +7129,33 @@ export class TextGeometry extends ExtrudeGeometry {
         bevelEnabled: boolean;
         bevelThickness: number;
         bevelSize: number;
+        bevelSegments: number;
+    };
+}
+
+export interface TextBufferGeometryParameters {
```

Comment:
It seems `TextBufferGeometryParameters` is identical to `TextGeometryParameters`, yes? Perhaps we could just reuse that interface?

Reply:
They are the same, ok, I will reuse the `TextGeometryParameters`.